packages/office-ui-fabric-react/src/components/Keytip/examples/Keytips.Basic.Example.tsx

```diff
@@ -7,32 +7,41 @@ import { Link } from 'office-ui-fabric-react/lib/Link';
 import { SpinButton } from 'office-ui-fabric-react/lib/SpinButton';
 import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
 import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+
+const pivotItemStyle: React.CSSProperties = { width: 500, paddingTop: 20 };
 
 export class KeytipsBasicExample extends React.Component<{}> {
   private _sampleOptions = [{ key: 'A', text: 'Option 1' }, { key: 'B', text: 'Option 2' }, { key: 'C', text: 'Option 3' }];
 
-  /* tslint:disable:jsx-ban-props jsx-no-lambda */
+  /* tslint:disable:jsx-no-lambda */
   public render(): JSX.Element {
     return (
       <div>
         <p>For Pivots, keytips will first show for each of the pivots. After selecting a pivot, the Keytips for its content are shown.</p>
         <Pivot>
-          <PivotItem headerText="Pivot 1" keytipProps={keytipMap.Pivot1Keytip} style={{ width: 500, paddingTop: 20 }}>
-            <SpinButton className="u-marginBottom" label={'Spin Button'} keytipProps={keytipMap.SpinButtonKeytip} />
-            <Toggle className="u-marginBottom" keytipProps={keytipMap.ToggleKeytip} onText={'Yes'} offText={'No'} />
-            <p>
-              Go to{' '}
-              <Link keytipProps={keytipMap.LinkKeytip} href={'http://www.bing.com'} target="_blank">
-                Bing
-              </Link>
-            </p>
+          <PivotItem headerText="Pivot 1" keytipProps={keytipMap.Pivot1Keytip} style={pivotItemStyle}>
+            <Stack gap={20}>
+              <SpinButton label="Spin Button" keytipProps={keytipMap.SpinButtonKeytip} styles={{ root: { maxWidth: 200 } }} />
```

Comment:
Why did you add the `styles={{ root: { maxWidth: 200 } }}` definition here?

Reply:
Because a 400px wide SpinButton is ugly.
![image](https://user-images.githubusercontent.com/5864305/55035687-cef06480-4fd5-11e9-9651-221d7be7d321.png)